xml/admin_docupdates.xml

```diff
@@ -728,7 +728,7 @@
    </varlistentry>
    <varlistentry>
 <!-- 28. File Synchronization -->
-    <term><citetitle>Available Data Synchronization Software</citetitle><!--<xref linkend="sec.net.sync.soft"/>-->
+    <term><citetitle>Available Data Synchronization Software</citetitle><!--<xref linkend="sec.net.rsync.soft"/>-->
```

Comment:
commented xref can be removed entirely, the linked section has been completely removed from the document